packages/blockchain-environment-v1/src/FabricEnvironmentConnection.ts

```diff
@@ -131,7 +161,7 @@ export class FabricEnvironmentConnection implements IFabricEnvironmentConnection
         }
     }
 
-    public async getCommittedSmartContracts(peerNames: Array<string>, channelName: string): Promise<Array<FabricCommittedSmartContract>> {
+    public async getCommittedSmartContracts(peerNames: Array<string>, channelName: string): Promise<Array<FabricSmartContractDefinition>> {
```

Comment:
Is it worth changing this function name to `getCommittedDefinitions`?

Reply:
ok